contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/mir/jobs.pxu

```diff
@@ -18,9 +18,10 @@ plugin: shell
 flags: also-after-suspend
 user: root
 depends: mir/check-ubuntu-frame-launching-auto
-imports: 
+imports:
     from com.canonical.certification import snap
     from com.canonical.plainbox import manifest
+    from com.canonical.certification import connections
```

Comment:
do we really need the connections resource here?

Reply:
humm no we don't. I missed this up. Will push the fix. Thanks.

Reply:
pushed the fix.